ansible/playbooks/jenkins/worker/create.yml

```diff
@@ -76,30 +76,30 @@
 # Ensure node is not installed anywhere but the linter servers
 #
 
-- hosts:
-  - test
-  - release
-  - "!test-packetnet-ubuntu1804-x64-1"
-  - "!test-packetnet-ubuntu1804-x64-2"
-  - "!test-ibm-ubuntu1804-x64-1"
-  tasks:
-    - name: remove node and npm packages
-      when: not os|startswith("win") and not os|startswith("zos") and not os|startswith("ibmi")
-      package:
-        name: "{{ package }}"
-        state: absent
-      loop_control:
-        loop_var: package
-      with_items: [ "node", "nodejs", "npm" ]
-    - name: fail if node is in the path - please uninstall it
-      when: not os|startswith("win")
-      shell: "node -v"
-      register: node_check_cmd
-      failed_when: node_check_cmd.rc == 0
-      changed_when: False
-    - name: fail if node is in the path - please uninstall it
-      when: os|startswith("win")
-      win_shell: "node -v"
-      register: node_check_cmd_win
-      failed_when: node_check_cmd_win.rc == 0
-      changed_when: False
+# - hosts:
```

Comment:
Delete instead of commenting out?

Reply:
ive fixed this and will push up the commit soon